src/components/Card.js

```diff
@@ -1,9 +1,23 @@
 export default class Card {
-  constructor( data, cardSelector, handleImageClick) {
+  constructor(
```

Comment:
Needs correction: when I click on the like button the delete confirmation appears, it should not be so, it should happen when you click on the trash button